_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,236 @@
+---
+title: 'Bitcoin Optech Newsletter #103'
+permalink: /en/newsletters/2020/06/24/
+name: 2020-06-24-newsletter
+slug: 2020-06-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a newly published fee ransom attack against
+LN users, links to continued discussion about the attack against LN
+atomicity, and shares a reminder about collision attacks on
+RIPEMD160-based addresses in multiparty protocols.  Also included are
+our regular sections with popular questions and answers from the Bitcoin
+StackExchange, a list of releases and release candidates published this
+week, and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **LN fee ransom attack:** René Pickhardt [publicly
+  disclosed][pickhardt post] a vulnerability to the Lightning-Dev
+  mailing list that he had previously privately disclosed to LN
+  implementation maintainers almost a year ago.  In the current LN
+  protocol, the party who initiates opening a channel must commit to
+  paying any onchain transaction fees that will be required to later
+  close that channel.  The party paying the fees also gets to choose the
+  feerate used---but the security of both parties depends on choosing an
+  appropriate feerate, so the counterparty may close the channel if they
+  think the chosen feerate is too low.  To avoid such unnecessary
+  closes, [BOLT2][] recommends the fee-paying party commit to fees five
+  times higher than the minimum reasonable estimate---high enough to
+  ensure the counterparty will be satisfied even if they're using a
+  different fee estimation algorithm.
+
+    BOLT2 also allows channels to route up to 483 payments
+    simultaneously, each of which requires a 43 vbyte P2WSH output, for
+    a total of about 20,000 vbytes of data that needs to be added to the
+    chain relatively quickly---meaning it may need to pay a high feerate.
+    If that feerate is five times higher than strictly necessary, this
+    can easily result in paying more than a hundred dollars in
```

Comment:
a sum described in dollars (USD? there are many different currencies called dollars) may not age well